src/System.ServiceModel.Duplex/pkg/System.ServiceModel.Duplex.pkgproj

```diff
@@ -20,6 +20,8 @@
     <InboxOnTargetFramework Include="xamarinmac20" />
     <InboxOnTargetFramework Include="xamarintvos10" />
     <InboxOnTargetFramework Include="xamarinwatchos10" />
+    <InboxOnTargetFramework Include="$(UAPvNextTFM)" />
```

Comment:
This puts placeholders in your package for UAP.  Can you confirm that this is indeed what you want?

Reply:
we said earlier that we instead wanted to still treat this as out of box to be able to service it separately, are we sure we don’t want this?

Reply:
@zhenlan and I have been discussing and still don't completely understand this whole issue. Could you give us your recommendations please. As it currently is this PR is failing in VSO...

https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_build?_a=summary&buildId=1487178

There is some issue with reference and implementation assemblies for uap10.0.16300. Adding it as InboxOnTargetFramework in the pkg project doesn't solve it.

Reply:
In uwp6.1 branch you need to update your package index to correctly represent what you shipped in uwp6.0.  You should leave the packages suppressing TreatAsOutOfBox on UAP.  That way you can continue to build these packages as standalone and allow folks to get a new version of WCF by getting a newer package.